src/components/Article.jsx

```diff
@@ -1 +1,31 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = (props) => {
+  const {
+    title,
+    date,
+    text,
+  } = props;
+
+  return (
+    <>
+      <h2 className="title" data-cy="title">
+        {title}
+      </h2>
+
+      <p className="subtitle">
+        <time dateTime={date.toISOString().slice(0, 10)} data-cy="date">
+          {date.toLocaleDateString('en-US', {
+            year: 'numeric',
+            month: 'long',
+            day: 'numeric',
+          })}
```

Comment:
it is better to move the calculation logic to the component itself, not to do it inside the return.